CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.6.2 (Under development)
+
+### App Center
+
+#### iOS
+
+* **[Fix]** Fix an error where **appcenter.podspec.json** could not be found.
+* **[Fix]** Fix issues with `use_frameworks!` directive.
```

Comment:
What kind of tests did we do for those 2 fixes?

Reply:
Regarding the renaming podspec bugfix, please test with cocoapods v1.7.5 and v1.8.4.

Regarding the use_frameworks bugfix, we probably need a repro of the bug and then verify this PR fixes the issue.

Reply:
@annakocheshkova please reply what you already tested for `use_frameworks!`

Reply:
Hey, I tested this on rn 0.59 and 0.61. Haven't tested with different CocoaPods versions though.

Reply:
@dhei I tested the change with cocoapods v.1.7.5 && v.1.8.4, works fine.